kcli/add-slb.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+set -xe
+
+oc apply -f add-slb-nncp.yaml
+./kcli/update-keepalived.sh brcnv-iface
```

Comment:
I'm surprised. Is not the keepalived service able to recognize the change on its own? I'm worried because of VIPs are used even in production clusters.

Reply:
at least the kcli is not, maybe a more advanced installation is able to detect the change are reconfigure the keepalived.

Reply:
Let's see what happens on Kobi's cluster. As long as we make sure that this command does not make it to production, it's ok